components/Staking/core/StakeTab.tsx

```
@@ -102,96 +110,114 @@ export function StakeTab({
+          updatedProgress.currentStepIndex = 2;
+          updatedProgress.steps[2].status = "success"; // Confirmation
+          updatedProgress.currentStepIndex = 3;
+          updatedProgress.steps[3].status = "processing"; // Now relaying
+          updatedProgress.overallStatus = "relaying";
+          break;
+
+        case "error":
+          // Find the current processing step and mark it as error
+          const processingIndex = updatedProgress.steps.findIndex(
+            (step) => step.status === "processing",
+          );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)